transmart-data/test_studies/Makefile

```diff
@@ -0,0 +1,15 @@
+SUBDIRS := $(wildcard */.)
+
+TRANSMART_COPY_VERSION := 17.1-HYVE-5.7
+TRANSMART_COPY := transmart-copy-$(TRANSMART_COPY_VERSION).jar
+TRANSMART_COPY_SRC_URL := https://repo.thehyve.nl/content/repositories/releases/org/transmartproject/transmart-copy/$(TRANSMART_COPY_VERSION)/transmart-copy-$(TRANSMART_COPY_VERSION).jar
```

Comment:
Well, the source is located at `../../transmart-copy`...

Reply:
true, but we release `transmart-data` as a separate component as well.

In dev setup, you could always build `transmart-copy` from sources and copy to the folder. `make load` won't fetch remote one if the version is right (`TRANSMART_COPY_VERSION `)